cdap-api-common/src/main/java/co/cask/cdap/api/data/format/FormatSpecification.java

```diff
@@ -37,12 +38,16 @@ private FormatSpecification() {
     this(null, null, Collections.<String, String>emptyMap());
   }
 
-  public FormatSpecification(String name, Schema schema, Map<String, String> settings) {
+  public FormatSpecification(String name, Schema schema, @Nullable Map<String, String> settings) {
```

Comment:
It was nullable before, but not annotated. I'd prefer to leave it as is for this PR.